src/rest_api/classes/transposon/widgets/feature.clj

```diff
@@ -0,0 +1,23 @@
+(ns rest-api.classes.transposon.widgets.feature
+  (:require
+    [datomic.api :as d]
+    [rest-api.classes.feature.core :as feature]
+    [rest-api.classes.generic-fields :as generic]))
+
+(defn associated-features [transposon]
+  (let [db (d/entity-db transposon)
+        data  (->>
+                (d/q '[:find [?f ...]
+                       :in $ ?transposon
+                       :where
+                       [?fg :feature.associated-with-transposon/transposon ?transposon]
+                       [?f :feature/associated-with-transposon ?fg]]
+                   db (:db/id transposon))
+                (map (partial feature/associated-feature db))
+                (seq))]
+    {:data (not-empty data)
```

Comment:
No need for `not-empty` since `seq` jas been applied already

Reply:
I just tested this and is seams to be true.